edit/security_functions.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * Verifies the username and password. If it is correct, any session data relating to incorrect entry is removed.
+ *
+ * @param string $username The given username.
+ * @param string $password The given password.
+ *
+ * @return bool Returns TRUE if the credentials match, otherwise FALSE.
+ */
+function verifyLogin($username, $password) : bool {
+    session_start();
+    if (loginExists($username)) {
+        if (correctPassword($username, $password)) {
+            unset($_SESSION['badname']);
+            unset($_SESSION['badpass']);
+            return true;
+        }
+    }
+    return false;
+}
+
+/*
+ * If the cookie ID does not match the session ID, the user is punted out of the session. GET OUTTA HERE
+ */
+function verifyUser() {
+    session_start();
+    if ($_COOKIE['id'] != session_id()) {
```

Comment:
this is not needed, just check a key in the session

Reply:
doesnt actually add any real security for session id stealing as its easy to fake a cookie